packages/react/index.js

```diff
@@ -10,16 +10,21 @@ module.exports = (neutrino, opts = {}) => {
   const mode = neutrino.config.get('mode');
   const options = merge({
     hot: true,
-    hotEntries: [require.resolve('react-hot-loader/patch')],
```

Comment:
There are now no default `hotEntries` in any of the presets that inherit `web` (or in `web` itself). Should we now remove `hotEntries` from the web preset entirely?

Reply:
I'm thinking so.